Dockerfile.base

```diff
@@ -0,0 +1,33 @@
+FROM python:2.7-buster
+
+# Install packages needed in Dockerfile
+RUN apt-get update && \
+   apt-get install -y sudo && \
+   apt-get install -y wget && \
+   apt-get install -y nodejs && \
+   apt-get install -y npm && \
+   apt install -y tzdata && \
+   apt-get clean
+
+# Create oppia directory in Docker container
+RUN mkdir /home/oppia
+
+# Install prerequisites. The yes package responds "yes" to all prompts.
+COPY ./scripts/install_prerequisites.sh /home/oppia/scripts/
+RUN bash /home/oppia/scripts/install_prerequisites.sh
+
+# Install chrome.
+COPY ./scripts/install_chrome.sh /home/oppia/scripts/
+RUN bash /home/oppia/scripts/install_chrome.sh
+ENV CHROME_BIN /usr/bin/google-chrome
+RUN export CHROME_BIN 
+
+RUN pip install enum34==1.1.10 protobuf==3.13.0 wheel==0.35.0 codecov
```

Comment:
Can we pin the codecov version? We recently had a pretty serious bug that broke all CI builds because of dependencies not being pinned

Reply:
Yes. What version should it be pinned to?

Reply:
Let's go with the most recent version since that's what this script does now